apps/prairielearn/src/tests/sharedQuestionPreview.test.ts

```diff
@@ -82,11 +81,32 @@ describe('Shared Question Preview', function () {
       isStudentPage: false,
     };
 
-    testQuestionPreviews(previewPageInfo, addNumbers, addVectors);
-
-    testFileDownloads(previewPageInfo, downloadFile, false);
-
-    testElementClientFiles(previewPageInfo, customElement);
+    describe('When questions are share_source_publicly but not shared_publicly', async function () {
+      before(
+        'Make sure question have share_source_publicly set but not shared_pubilcly',
```

Comment:
"... question have ..." is improper grammar. "pubilcly" is spelled wrong.

Reply:
fixed